### PR TITLE
Update Cordova to 10.0.0

### PR DESCRIPTION
**Pull request by costinelmarin**

### CODE REVIEW

**Comment by renanccastro** [comment]
Thanks for this! Awesome work!
The tests are outputting the following when creating new Cordova apps:
"Error: Cannot find module 'properties-parser'"
We are probably missing dependencies from cordova-create, since they have split the cordova-lib.

**Comment by Nauzer** [comment]
Great work!
How can I help in getting this production ready?

**Comment by costinelmarin** [comment]
I was able to replicate the issue locally and push an update.

File: tools/cordova/index.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ export const CORDOVA_PLATFORMS = ['ios', 'android'];
 export const CORDOVA_DEV_BUNDLE_VERSIONS = {
   'cordova-lib': '10.0.0',
   'cordova-common': '4.0.2',
-  'cordova-create': '3.0.0',
+  'cordova-create': '2.0.0',
```
Comment (renanccastro) [inline tools/cordova/index.js:19]: Is it a downgrade @costinelmarin? if we don't have a strong reason to downgrade, we probably should not.

**Review by renanccastro — CHANGES_REQUESTED** [review]

File: tools/cordova/index.js (hunk 1 above)
Comment (costinelmarin) [reply tools/cordova/index.js:19]: @renanccastro the android platform was not working with `'cordova-create': '3.0.0'`. I was able to make it work locally by using `2.0.0` version.

Comment (renanccastro) [reply tools/cordova/index.js:19]: why wasn't it working? maybe we have another solution.

Comment (costinelmarin) [reply tools/cordova/index.js:19]: Some of the dependency were not installed, `properties-parser` package and when adding android platform I get this message `%% cordova-android version check failed (project/.meteor/local/cordova-build/platforms/android/cordova/version), continuing anyways.`.

**Comment by costinelmarin** [comment]
@renanccastro can we get more details why the travis build failed? 
Thanks!

**Comment by filipenevola** [comment]
@costinelmarin the problem on Travis was a timeout, I'm running it again.

**Comment by costinelmarin** [comment]
Hey @filipenevola. It is possible to run the Travis again and release this in the next meteor version if the tests pass? 
Thanks!

**Review by renanccastro — APPROVED** [review]

**Comment by shahdeep1989** [comment]
@renanccastro  &  @costinelmarin 
Can I expect this changes in next meteor release in meteor 2.2? If yes when?

**Comment by mjcctech** [comment]
is launchScreens [https://docs.meteor.com/api/mobile-config.html#App-launchScreens] also updated to cordova-10? I have issue with iOS as configuration has changed [https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-splashscreen/index.html#ios-specific-information].

**Pull request merged**